.ci/shippable.sh

```diff
@@ -131,6 +131,14 @@ validate-ci-temp-empty)
   exit $fail
   ;;
 
+git-status)
+  if [ $(git status | grep "Changes not staged for commit" | wc -l) -gt 0 ]; then
+    echo "There are changes in files after clone, recheck .gitattributes file"
+    sleep 5s
```

Comment:
Please print git status before sleep, to show contributors what is left

Reply:
Fixed.